### PR TITLE
Swap out axios for @digitalbazaar/http-client.

### PR DESCRIPTION
**Pull request by aljones15**
Patches the intermittent release in master with the removal of axios.

1. Replaces axios in `getEventStream` API
2. Replaces axios in `getHistory` API
3. Replaces axios in `notifyPeer` API
4. All methods are called by the existing tests
5. All methods have tests for success in the tests.
6. Streams from `getEventStream` are being read.
5. `getHistory` & `notifyPeer` both have negative tests which hit the new error handler.
6. `getEventStream` returns a `ReadableStream` which is in a readable state (has not been used yet)
7. `getEventStream` does not have a test testing failure, but `response.ok` is on the ky response object.

### CODE REVIEW

File: CHANGELOG.md
Comment (mandyvenables) [inline CHANGELOG.md:12]: I would think that just
```suggestion
### Changed
- Replaced `axios` with `@digitalbazaar/http-client`
```
would suffice, but unsure if others think the details are more useful.

**Review by mandyvenables — APPROVED** [review]

File: CHANGELOG.md
Comment (aljones15) [reply CHANGELOG.md:12]: @mandyvenables I'll let @mattcollier decide on this. He had issues with my CHANGELOG notes in a previous PR.

**Review by mattcollier — CHANGES_REQUESTED** [review]

File: lib/client.js
Comment (aljones15) [reply lib/client.js:129]: it's a peerDep and set `2.0.0`: https://github.com/digitalbazaar/bedrock-ledger-consensus-continuity/blob/710c34ee88e87b544339ddebe9a3576bc7579fd9/package.json#L45-L49

**Review by dlongley — APPROVED** [review]
Thanks!

**Comment by aljones15** [comment]
- [ ] note: this will be tested by @gannan08 to see performance differences between axios and `@digitalbazaar/http-client`

DO NOT MERGE YET

**Comment by gannan08** [comment]
> * [ ]  note: this will be tested by @gannan08 to see performance differences between axios and `@digitalbazaar/http-client`
> 
> DO NOT MERGE YET

No regression in performance detected. Ledger performance is nominal.

**Pull request merged**
